assets/src/edit-story/components/panels/panel/title.js

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useContext, useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useInspector from '../../inspector/useInspector';
+import panelContext from './context';
+import DragHandle from './handle';
+import Arrow from './arrow.svg';
+
+const Header = styled.h2`
+	background-color: ${ ( { theme, isPrimary } ) => isPrimary ? theme.colors.fg.v2 : theme.colors.fg.v1 };
+	border: 0 solid ${ ( { theme } ) => theme.colors.fg.v2 };
+	border-top-width: ${ ( { isPrimary } ) => isPrimary ? 0 : '1px' };
+	color: ${ ( { theme } ) => theme.colors.bg.v2 };
+	padding: 10px 20px;
```

Comment:
Instead of putting the padding here, can we put it on the button [here](https://github.com/google/web-stories-wp/pull/143/files#diff-382574245b322d4af89b8c10d7437990R55).

Reply:
This actually follows the [W3 guideline on accordions](https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html) 1-to-1. I have no opinion one way or the other - just followed their lead.